src/main/java/bio/terra/common/sam/exception/SamTimeoutException.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.common.sam.exception;
+
+import bio.terra.common.exception.InternalServerErrorException;
+
+/**
+ * This exception is thrown when a request fails due to a timeout. Timeouts are generally retried,
+ * but this may be thrown if repeated retries fail.
+ */
+public class SamTimeoutException extends InternalServerErrorException {
```

Comment:
I think this a 503 instead of a 500 case, but I'm not an http code expert, so use your judgment.